internal/blob/downloader.go

```diff
@@ -37,6 +38,9 @@ func NewDownloader(logger logging.Logger, baseCacheDir string) *downloader { //n
 }
 
 func (d *downloader) Download(ctx context.Context, pathOrURI string) (Blob, error) {
+	pathComponents := strings.Split(pathOrURI, "/")
+	buildpackRef := fmt.Sprintf("%s/%s", pathComponents[len(pathComponents)-2], pathComponents[len(pathComponents)-1])
```

Comment:
@natalieparellano sorry for being this late.
This logic here was used to create the buildpack name for using in the log message since I was not able to fetch the name within the function. I assumed that the last two elements of the path would always represent the buildpack id.

Reply:
Ah, apologies for not being more clear - here I **think** that `pathOrURI` is the user-provided value, whereas `parsedURL` is the fully parsed value. So we'd want to log something like `Using <pathOrURI> found at <parsedURL>`.